tasks/src/unconditional-image-generation/about.md

```diff
@@ -0,0 +1,19 @@
+## About the Task
+
+Unconditional image generation is the task of generating images with no condition in any context (like a *prompt text* or *another image*). Once trained, the model will create images that resemble its training data distribution.
+Unconditional image generation models usually start with a *seed* that generates a *random noise vector*. The model will then upsample this vector to create the final image. 
+An example of unconditional image generation would be generating the image of a face on a model trained with the [CelebA dataset](https://huggingface.co/datasets/huggan/CelebA-HQ).
+
+## Use Cases
```

Comment:
Can you leave these sections descriptions as well so it's easier for others to contribute🙂